lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -1,45 +1,91 @@
-<div class="adf-metadata-properties">
-    <mat-accordion displayMode="flat"
-                   [multi]="multi">
-        <mat-expansion-panel *ngIf="displayDefaultProperties"
-                             [expanded]="canExpandProperties()"
-                             [attr.data-automation-id]="'adf-metadata-group-properties'">
-            <mat-expansion-panel-header>
-                <mat-panel-title class="adf-metadata-properties-title">
-                    {{ 'CORE.METADATA.BASIC.HEADER' | translate }}
-                </mat-panel-title>
-            </mat-expansion-panel-header>
-            <adf-card-view
-                (keydown)="keyDown($event)"
-                [properties]="basicProperties$ | async"
-                [editable]="editable"
-                [displayEmpty]="displayEmpty"
-                [copyToClipboardAction]="copyToClipboardAction"
-                [useChipsForMultiValueProperty]="useChipsForMultiValueProperty"
-                [multiValueSeparator]="multiValueSeparator">
-            </adf-card-view>
-        </mat-expansion-panel>
-        <ng-container *ngIf="displayTags">
-            <mat-expansion-panel *ngIf="!editable">
-                <mat-expansion-panel-header>
-                    <mat-panel-title>{{ 'METADATA.BASIC.TAGS' | translate }}</mat-panel-title>
-                </mat-expansion-panel-header>
-                <p *ngFor="let tag of tags" class="adf-metadata-properties-tag">{{ tag }}</p>
-            </mat-expansion-panel>
-            <div
-                *ngIf="editable"
-                class="adf-metadata-properties-tags">
-                <div class="adf-metadata-properties-tags-title">
-                    <p>{{ 'METADATA.BASIC.TAGS' | translate }}</p>
-                    <button
-                        data-automation-id="showing-tag-input-button"
+<mat-accordion displayMode="flat" [multi]="multi" class="adf-metadata-properties">
+    <mat-expansion-panel
+        *ngIf="displayDefaultProperties"
+        [(expanded)]="isGeneralPanelExpanded"
+        [attr.data-automation-id]="'adf-metadata-group-properties'"
+        hideToggle>
+        <mat-expansion-panel-header>
+            <adf-content-metadata-header [title]="'CORE.METADATA.BASIC.HEADER'" [expanded]="isGeneralPanelExpanded">
+                <button *ngIf="canEditGeneralInfo"
                         mat-icon-button
-                        [attr.title]="'METADATA.BASIC.ADD_TAG_TOOLTIP' | translate"
```

Comment:
we can remove translation key 'METADATA.BASIC.ADD_TAG_TOOLTIP' from en.json file